crypto/sha/asm/sha1-x86_64.pl

```diff
@@ -352,6 +368,7 @@ sub BODY_40_59 {
 .align	32
 sha1_block_data_order_shaext:
 _shaext_shortcut:
+.cfi_startproc
```

Comment:
I'm kind of leaning toward letting be annotating frame-less subroutines, i.e. those that require .cfi_{start|end}proc alone. It does work, i.e. debugger does manage frame-less unannotated correctly. Verified in #2586 context. But don't bother removing them here, I mean since there are added already...

On side note, I've considered adding .cfi_{start|end}proc automatically, but decided that it's not the time. It would be appropriate to consider it only after all code is annotated, because it's probably hardly appropriate to auto-add start/end and *not* add other annotations if subroutine is not frame-less...